themes/dg-theme/assets/css/main.scss

```diff
@@ -7,6 +7,11 @@ $screenheight-short: 900px;
 a, a:hover, a:visited, a:active {
   color: inherit;
   text-decoration: none;
+  white-space: pre-wrap; /* CSS3 */    
+  white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
+  white-space: -pre-wrap; /* Opera 4-6 */    
+  white-space: -o-pre-wrap; /* Opera 7 */    
```

Comment:
I think these are unnecessary, just using the `word-wrap` is enough to get the links to wrap properly on all modern browsers. In fact these rules break the styling in a few unrelated places (the "Select your spec" section on the front page no longer shows the text, the date line under each post title on the list pages has a bunch of extra whitespace now), so I suggest we just drop them entirely.

```suggestion
```

Reply:
Yup, Oi made me aware of that. I'm curious as to why this actually breaks links 

(I think it happens because the link contains <img/> and <p/> and hence wraps after the image. Setting display:flex on it actually seems to solve the issue)

The fact that legacy rules break stuff on modern browsers makes me a bit wary but I'm no CSS expert so I'll trust your judgement